src/include/OpenImageIO/simd.h

```diff
@@ -5003,6 +4980,7 @@ OIIO_FORCEINLINE vint4 rotl(const vint4& x, int s) {
 }
 
 // DEPRECATED (2.1)
+OIIO_DEPRECATED("Use rotl32 instead")
```

Comment:
Typo? Should this be `rotl` instead of `rotl32`?
```suggestion
OIIO_DEPRECATED("Use rotl instead")
```

Reply:
quite right, thanks